qa/tasks/cephfs/test_volume_client.py

```diff
@@ -1078,7 +1077,7 @@ def test_volume_without_namespace_isolation(self):
         volume_prefix = "/myprefix"
         group_id = "grpid"
         volume_id = "volid"
-        mount_path = self._volume_client_python(vc_mount, dedent("""
+        _ = self._volume_client_python(vc_mount, dedent("""
```

Comment:
if the right part needed, then maybe just remove left part of?

Reply:
Done